tokio-rustls/src/lib.rs

```diff
@@ -315,6 +315,22 @@ impl<IO> Accept<IO> {
     pub fn into_fallible(self) -> FallibleAccept<IO> {
         FallibleAccept(self.0)
     }
+
+    pub fn get_ref(&self) -> Option<&IO> {
```

Comment:
for symmetry, can we also add it to `Connect`?

Reply:
Done.